python/pyiceberg/table/metadata.py

```diff
@@ -84,12 +85,12 @@ class TableMetadataCommonFields(IcebergBaseModel):
     def current_schema(self) -> Schema:
         return next(schema for schema in self.schemas if schema.schema_id == self.current_schema_id)
 
-    @root_validator(pre=True)
+    @root_validator()
     def cleanup_snapshot_id(cls, data: Dict[str, Any]):
-        if data.get("current-snapshot-id") == -1:
+        if data.get("current_snapshot_id") == -1:
```

Comment:
Can the changes to remove prevalidators be done in a separate PR?

Reply:
I've removed those 👍🏻